common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1859,23 +1868,43 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
-  def columnarShuffleEnabled(conf: CelebornConf): Boolean = {
-    conf.getBoolean("rss.columnar.shuffle.enabled", defaultValue = false)
-  }
+  val COLUMNAR_SHUFFLE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.columnar.shuffle.enabled")
+      .categories("columnar-shuffle")
+      .version("0.2.0")
+      .doc("Whether to enable columnar-based shuffle.")
+      .booleanConf
+      .createWithDefault(false)
 
-  def columnarShuffleCompress(conf: CelebornConf): Boolean = {
-    conf.getBoolean("rss.columnar.shuffle.encoding.enabled", defaultValue = false)
-  }
+  val COLUMNAR_SHUFFLE_DICT_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.columnar.shuffle.dictionary.enabled")
+      .categories("columnar-shuffle")
+      .version("0.2.0")
+      .doc("Whether to use dictionary encoding for columnar-based shuffle data.")
```

Comment:
I'm not sure about the design purpose of this configuration, the description from my understanding of the code.